model/lesson.js

```diff
@@ -122,6 +139,36 @@ async function getLessonById(data) {
     );
 }
 
+/*Fetches verses in a lesson based*/
+//ASSUMPTIONS: Verses in a Lesson are contigous
+async function getLessonVerses(data) {
+
+    let lesson = await getLessonById(data);
+    if (!lesson.success) {
+        return lesson;
+    }
+
+    let lessonInfo = [];
+    let currentVerse = lesson.data[0].start_verse;
```

Comment:
In the case that start_verse is null, what do we do?
How about:
`let currentVerse = lesson.data[0].start_verse || 1;`

But what would we do for the end verse? --> Maybe we can call getVerseInfo up until it tells us that we get an error

Reply:
sent screenshot through slack.

Reply:
ya, I think I understand the issue. Will cover this case. Thanks